SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/AffixEmpyrean.cs

```diff
@@ -1,20 +1,267 @@
-﻿using RoR2;
+﻿using MSU;
+using RoR2;
+using RoR2.ContentManagement;
+using System;
+using System.Collections;
 using System.Collections.Generic;
+using UnityEngine;
+using MonoMod.Cil;
+using Mono.Cecil.Cil;
+using System.Linq;
 namespace SS2.Equipments
 {
-    //[DisabledContent]
-    /*public sealed class AffixEmpyrean : SS2EliteEquipment
+    public sealed class AffixEmpyrean : SS2EliteEquipment
     {
-        public override EquipmentDef EquipmentDef { get; } = SS2Assets.LoadAsset<EquipmentDef>("AffixEmpyrean", SS2Bundle.Equipments);
+        public override EquipmentDef EquipmentDef => _equipmentDef;
+        private EquipmentDef _equipmentDef;
+        public override List<EliteDef> EliteDefs => _eliteDefs;
+        private List<EliteDef> _eliteDefs;
 
-        public override List<MSEliteDef> EliteDefs { get; } = new List<MSEliteDef>
+        private Material matOverlay;
+
+        private GameObject affixEffect;
+
+        public override NullableRef<List<GameObject>> ItemDisplayPrefabs => null;
+
+        public static List<EliteDef> blacklistedEliteDefs = new List<EliteDef>();
+
+        public override void Initialize()
+        {
+            BuffOverlays.AddBuffOverlay(SS2Content.Buffs.bdEmpyrean, matOverlay);
```

Comment:
Adding buff overlays is unecesary as the EquipmentModule already adds the overlays by using the one found in the extendedEliteDef